src/styles/header/catalog.scss

```diff
@@ -0,0 +1,21 @@
+.catalog {
+  display: flex;
+  padding: 50px 128px;
+  flex-wrap: wrap;
+  justify-content: center;
+
+  &__first-block {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__second-block {
+    display: flex;
+    justify-content: space-between;
+  }

```

Comment:
Don't repeat yourself. Your `first-block` and `second-block` classes are the same.
Also, you may avoid creating them at all.
What if there will be three or more rows? You can just limit the max-width of your container (`.catalog` class) and all cards will be transferred to the new lines if there will be more than 4 of them in row